internal/topo/planner/analyzer.go

```diff
@@ -265,12 +375,13 @@
 
 type fieldsMap struct {
 	content       map[string]streamFieldStore
+	alias         map[string]struct{}
```

Comment:
The content should also have the alias information. Is it worth to reuse it?

Reply:
We need to collect the aliasName first before bind the schema info for column if they reference the alias, then we can bind the alias info with column schema info.